src/MSBuild.SDK.SystemWeb.RazorLibrary/Sdk/MSBuild.SDK.SystemWeb.RazorLibrary.DefaultPackageControlledVersions.targets

```diff
@@ -0,0 +1,100 @@
+﻿<!-- 
```

Comment:
This file should probably be a common file now, and use the `ExcludeSDKDefaultPackages` and `ApplySDKDefaultPackageVersions` properties instead of `ExcludeDefaultRazorPackages` and `EnablePackageVersionDefaultsFromSdk`

Reply:
I'll try to take a look at it in the next several days.